### PR TITLE
RD facelift, sprite updates to the coat and turtleneck

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request
Tin.
RD pr

## How This Contributes To The Skyrat Roleplay Experience

Sprites

## Changelog

:cl:
imageadd: The RD's labcoat and turtleneck have been resprited
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 1]
```diff
@@ -8,8 +8,8 @@
 /obj/item/clothing/suit/toggle/labcoat/rd
 	name = "research directors labcoat"
 	desc = "A Nanotrasen standard labcoat for Research Directors. It has extra layers for more protection."
-	icon = 'modular_skyrat/modules/aesthetics/clothing/items.dmi'
-	worn_icon = 'modular_skyrat/modules/aesthetics/clothing/suit.dmi'
```
Comment (ghost) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm:12]: Make sure the icons are removed from these if you put them into master files. Wanna work to kill the duplicate clothing icon files we have scattered about.

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 2]
```diff
@@ -87,3 +87,16 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/shoes.dmi'
 	icon_state = "workbootsold"
+
+
+// UNDERSUITS
```
Comment (ghost) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
/* 
UNDERSUITS
*/
```
Was told this way better. Dunno if it's really true tbh BUT it makes it easier to find these while scrolling down anyways, so

**Comment by ghost** [comment]
🙏 checks out aside from the 2 comments
For now having the turtleneck overrides is fine but I do ask that those get sent upstream soon. They'd benefit from it, and if they want to change it (i.e. keep the tan pants and not your brown) then it's best we know that now/soon instead of by surprise later

**Comment by Ebin-Halcyon** [comment]
> 🙏 checks out aside from the 2 comments For now having the turtleneck overrides is fine but I do ask that those get sent upstream soon. They'd benefit from it, and if they want to change it (i.e. keep the tan pants and not your brown) then it's best we know that now/soon instead of by surprise later

They aren't doing the science resprite for a long awhile iirc, i'd just fix it down here until Viro or Imaginos do roll around to do it upstream and then i'll fix it down here.

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 3]
```diff
@@ -87,3 +87,18 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/shoes.dmi'
 	icon_state = "workbootsold"
+
+
+/* 
+UNDERSUITS
+*/
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
/*
*	UNDERSUITS
*/
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 4]
```diff
@@ -87,3 +87,18 @@
...
+
+/obj/item/clothing/under/rank/rnd/research_director/turtleneck/skirt
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/rnd.dmi'
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/rnd.dmi'
+
+
+/obj/item/clothing/under/rank/rnd/research_director/turtleneck
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/rnd.dmi'
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/rnd.dmi'
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
/obj/item/clothing/under/rank/rnd/research_director/turtleneck
	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/rnd.dmi'
	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/rnd.dmi'
	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION
	
/obj/item/clothing/under/rank/rnd/research_director/turtleneck/skirt
	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/rnd.dmi'
	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/rnd.dmi'
	
```

**Review by tf-4 — COMMENTED** [review]
just some minor styling stuff

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm (hunk 3 above)
Comment (ghost) [reply modular_skyrat/modules/aesthetics/clothing/clothing.dm]: I was close!

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by HoodMilk** [comment]
looks better 👍

**Review by tf-4 — APPROVED** [review]

**Pull request merged**